simSPI/linear_simulator/linear_simulator.py

```diff
@@ -0,0 +1,85 @@
+"""Contain simulator and its component classes."""
+
+import os
+
+import mrcfile
+import torch
+from compSPI.transforms import fourier_to_primal_2D, primal_to_fourier_2D
+
+from simSPI.linear_simulator.ctf import CTF
+from simSPI.linear_simulator.noise_utils import Noise
+from simSPI.linear_simulator.projector import Projector
+from simSPI.linear_simulator.shift_utils import Shift
+from simSPI.linear_simulator.volume_utils import init_cube
+
+
+class LinearSimulator(torch.nn.Module):
+    """Class to generate data using linear forward model.
```

Comment:
clarify in docstring somewhere about how projection done (real space)

Reply:
I think simulator should be agnostic to projector method. So we dont need to explain how the projection works.